schemas/com.iterable/system_webhook/jsonschema/1-0-0

```diff
@@ -0,0 +1,643 @@
+                  "number",
+                  "null"
+                ],
+                "minimum": 0,
+                "description": "Reference to the type of message the event is associated with (e.g., 14381)"
+              },
+              "messageTypeIds": {
+                "type": [
+                  "array",
+                  "null"
+                ],
+                "items": {
+                  "type": "number",
+                  "minimum": 0
+                },
+                "description": "Message type IDs"
+              },
+              "platformEndpoint": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "The specific platform endpoint of the event",
+                "maxLength": 4096
+              },
+              "productRecommendationCount": {
+                "type": [
+                  "number",
+                  "null"
+                ],
+                "description": "Product recommendation count."
+              },
+              "profileUpdatedAt": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "2020-03-20 23:11:58 +00:00",
+                "maxLength": 255
+              },
+              "proxySource": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "Proxy source (e.g., 'Apple')",
+                "maxLength": 4096
+              },
+              "pushMessage": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "Push message text"
+              },
+              "reason": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "Reason (e.g., 'DuplicateMarketingMessage')"
+              },
+              "recipientState": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "The state of the recipient upon receiving the event (e.g., 'HardBounce')",
+                "maxLength": 4096
+              },
+              "region": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "Region of the event (e.g., 'CA')",
+                "maxLength": 4096
+              },
+              "signupSource": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "Sign-up source (e.g., 'UpdateSubscriptionsAPI')",
+                "maxLength": 4096
+              },
+              "smsMessage": {
+                "type": [
+                  "string",
+                  "null"
+                ],
+                "description": "SMS message text"
```

Comment:
another one hard to restict, but maybe there is some limit for sms length? maybe something relatively big sounds safe? what do you think?